src/ripple/app/misc/detail/WorkSSL.h

```diff
@@ -25,7 +25,7 @@
 #include <ripple/core/Config.h>
 #include <ripple/net/HTTPClientSSLContext.h>
 #include <boost/asio/ssl.hpp>
-#include <boost/bind.hpp>
+#include <boost/bind/bind.hpp>
```

Comment:
This is the _only_ place where we call `boost::bind`. We should be able to just replace `boost::bind` with `std::bind` and remove the dependence.

Reply:
Great idea! Done (force pushed since it's such a small PR).